packages/fx-core/src/component/resource/aadApp/aadApp.ts

```diff
@@ -191,11 +183,6 @@ export class AadApp implements CloudResource {
     return res;
   }
 
-  @hooks([
```

Comment:
We still will call collaboration/buildAadManifest for non-lifecycle commands. Will this change have any influence?
cc @wenytang-ms

Reply:
No, because FxCore already has a middleware to replace this deleted one.

Reply:
I will make up for the telemetry by addStartAndEndTelemetry(actionName, actionName) middleware in another PR.